unreleased_history/public_api_changes/fix_tickers_typo.md

```diff
@@ -0,0 +1,4 @@
+Some tickers (namely, `ERROR_HANDLER_BG_ERROR_COUNT`, `ERROR_HANDLER_BG_IO_ERROR_COUNT`, and `ERROR_HANDLER_BG_RETRYABLE_IO_ERROR_COUNT`)
```

Comment:
nit: it would be great if this can be updated to refer to the publicly visible names of these tickers like `rocksdb.error.handler.bg.errro.count`, since `ERROR_HANDLER_BG_ERROR_COUNT` is the enum type RocksDB internally uses to refer to the ticker type. 

How about: 
Add new tickers: `rocksdb.error.handler.bg.error.count`, `rocksdb.error.handler.bg.retryable.io.error.count`, `rocksdb.error.handler.bg.error.count` to replace the misspelled ones: `rocksdb.error.handler.bg.errro.count`, `rocksdb.error.handler.bg.retryable.io.errro.count`, `rocksdb.error.handler.bg.errro.count`. Users should switch to use the new tickers before 9.0 release as the misspelled old tickers will be completely removed then.

We usually make this a one paragraph message so it shows up as one entry in HISTORY.md later when we make the release.